### PR TITLE
fix: fix lastValue return err in workspace overview page

### LINKED ISSUE #3737: JS err in workspace overview page
**General remarks**

> This form is to report bugs. For general usage questions refer to our Slack channel
>        [KubeSphere-users](https://join.slack.com/t/kubesphere/shared_invite/enQtNTE3MDIxNzUxNzQ0LTdkNTc3OTdmNzdiODViZjViNTU5ZDY3M2I2MzY4MTI4OGZlOTJmMDg5ZTFiMDAwYzNlZDY5NjA0NzZlNDU5NmY)

**Describe the bug**
url: workspaces/system-workspace/overview/usage

For UI issues please also add a screenshot that shows the issue.

**Versions used(KubeSphere/Kubernetes)**
KubeSphere: release-3.2
Kubernetes: (If KubeSphere installer used, you can skip this)

**Environment**
How many nodes and their hardware configuration: 

For example:
3 masters:  8cpu/8g
3 nodes: 8cpu/16g

(and other info are welcomed to help us debugging)

**To Reproduce**

**Expected behavior**

### PR DESCRIPTION
**Pull request by moweiwei**
Signed-off-by: moweiwei <mo.weiwei@99cloud.net>

### What type of PR is this?

/kind bug

### Which issue(s) this PR fixes:

Fixes #3737 

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: src/components/Charts/Area/MediumArea/index.jsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default class MediumArea extends React.Component {
   get lastValue() {
     const { data } = this.props
     const key = get(this.series, [0], '')
-    return last(data)[key] || 0
+    return (last(data) && last(data)[key]) || 0
```
Comment (harrisonliu5) [inline src/components/Charts/Area/MediumArea/index.jsx:63]: maybe `last(data)?.[key] ?? 0` is better?

Comment (moweiwei) [reply src/components/Charts/Area/MediumArea/index.jsx]: nice

**Comment by moweiwei** [comment]
@harrisonliu5

**Comment by harrisonliu5** [comment]
Thank you for your contribution!

**Pull request merged**